### ISSUE TITLE
Publish Low Resolution

### ISSUE THREAD

**Issue by krishnaavril** [ynput/ayon-nuke#41]
### Is there an existing issue for this?

- [x] I have searched the existing issues and added correct labels.

### Please describe the feature you have in mind and explain what the current shortcomings are?

Some projects will have more than 4k resolutions.

we probably don't need to work with project settings sometimes, especially when it's going for during the initial stage of shot. rendering the higher resolution effects the production time.

### Suggested implementation?

It would be great to have the exposed format parameter on the write node. let the user publish the previews at his won resolution.

The current validation is good for hinting the artist before publishing

### Describe alternatives you've considered:

I ran some tests on the above solution, correct me if this is already existed

**Comment by jakubjezek001** [comment]
Yes, this will need to be added. Thank you for the suggestion. We will need to make sure all validators will pass too. Currently, we are validating the Render product is settings the Shot attributes for output resolution.
